torchvision/utils.py

```diff
@@ -375,13 +375,7 @@ def _generate_color_palette(num_masks: int):
     return [tuple((i * palette) % 255) for i in range(num_masks)]
 
 
-@no_type_check
-def _log_api_usage_once(obj: str) -> None:  # type: ignore
+def _log_api_usage_once(module: str, name: str) -> None:
     if torch.jit.is_scripting() or torch.jit.is_tracing():
```

Comment:
I wonder if this is still necessary for JIT or if the if statement can be removed?

Reply:
It's still necessary, torch.jit.script complains
> Python builtin <built-in method _log_api_usage_once of PyCapsule object at 0x7f6b4d09a780> is currently not supported in Torchscript:
def foo(x: torch.Tensor) -> torch.Tensor:
    torch._C._log_api_usage_once("torchvision.hi")
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
    return x